lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -154,6 +154,38 @@ class CheckRepressions {
   }
 };
 
+/// If the extension adds a conformance to an invertible protocol, ensure that
+/// it does not add a conformance to any other protocol. So these are illegal:
+///
+///     extension S: Copyable & P {}
+///     extension S: Q, Copyable {}
+///
+/// This policy is in place because extensions adding a conformance to an
+/// invertible protocol do _not_ add default requirements on generic parameters,
+/// so it would be confusing to mix them together in the same extension.
+static void checkExtensionAddsSoloInvertibleProtocol(const ExtensionDecl *ext) {
+  InvertibleProtocolSet inverses;
+  bool anyObject;
+  auto inherited =
```

Comment:
Might be better to check getLocalConformances() instead

Reply:
yup this works and is much nicer.